en/docs/guides/identity-verification/add-idvp.md

```diff
@@ -0,0 +1,27 @@
+# Registering an identity verification provider
+
+WSO2 Identity Server allows you to add identity verification providers (IDVP) and specify various details that help you link the identity verification provider to the WSO2 Identity Server. 
+To properly configure the IDVPs, you must specify all information required to perform identity verification.
+
+This guide walks you through adding and configuring identity verification providers based on your requirements.
+
+!!! note
+Adding and configuring an IDVP can be performed by administrators only.
+
+## Register an identity verification provider
+
+To add a new identity verification provider.
+
+1. On WSO2 Identity Server Management Console, go to **Main > Identity > Identity Verification Providers**
+
+2. Click **Add**.  
+   ![add-idvp]({{base_path}}/assets/img/guides/add-idvp.png)
+
+3. Add Identity verification provider's name and description and select the required identity verification provider.
+
+4. Then add configurations of the identity verification provider.
```

Comment:
```suggestion
4. Add configurations of the identity verification provider.
```

Reply:
Shall we describe the parameter provided in the UI? `Token` and `Base URL`

Reply:
Regarding describing Token and Base URL, these configurations can vary based on the IDV provider. Therefore IMO, mentioning these two configurations will confuse the reader. In connector-level documents we can describe those configs.